CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Code of Conduct
+permalink: /code-of-conduct/
+---
+
+# Code of Conduct
+
+In order to foster a welcoming, positive, and rewarding experience, we as community members take an oath to uphold this code of conduct.
```

Comment:
I am a bit surprised our linter for markdown files didn't mark the document for using lines longer than 80 chars…

Reply:
That check might be disabled; will double check.  Regardless, I will hard-wrap this is 80.